tests/foreman/api/test_hostgroup.py

```diff
@@ -133,7 +115,7 @@ def test_inherit_puppetclass(self):
         )
         response.raise_for_status()
         results = response.json()['results']
-        puppet_class_id = results['ntp'][0]['id']
+        puppet_class_id = results['api_test_classparameters'][0]['id']
```

Comment:
Consider using the variable(s) you created instead of this string. `repo`/`puppet_class`